### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by carlosmuerto**
# Awesome books: refactor to use JavaScript classes

### Full [README](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md)

### CODE REVIEW

File: code.js [hunk 1]
```diff
@@ -68,33 +11,28 @@ const listOfBooksElement = document.querySelector('#list-books .list-of-books');
...
 function CreateBookItemHTML(id, title, author) {
   const divContainer = document.createElement('div');
-  const bookTitleElement = document.createElement('p');
-  const bookAuthorElement = document.createElement('p');
+  const bookTitleAndAuthorElement = document.createElement('p');
   const deleteBookBtn = document.createElement('button');
-  const hrElement = document.createElement('hr');
 
   divContainer.id = `Book-${id}`;
+  divContainer.classList.add('books-details');
```
Comment (BertrandConxy) [inline code.js:23]: - [x] I have seen that you have refactored the code in separate files which is good 👏 (i.e: in the js folder) and you kept the initial code in the `code.js` file and it is still referenced in your `index.html` file **while it contains the code with functions while we have to use classes**. However, I realized that the application is not working. I can't add the book and therefore I can't remove any. Let me show you the error being displayed in the browser console 👇 
image

**PLEASE reference your entry point js file that uses classes in your `index.html` and see if the application works fine. However, you can still work to make the application perform as expected**

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @carlosmuerto ✋, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 

## Highlights 💪 
- no linter errors
- good PR description
- good README


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: code.js (hunk 1 above)
Comment (carlosmuerto) [reply code.js:23]: fixed, this happens because local storage initialisation was not null safe.

**Review by Lordkaito — APPROVED** [review]
Hi @cmolinan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**